Makefile

```diff
@@ -268,8 +269,9 @@ upload-coverage: $(BOOTSTRAP_TARGET)
 	@build/upload-coverage.sh
 
 .PHONY: acceptance
+acceptance: TESTTIMEOUT := $(ACCEPTANCETIMEOUT)
```

Comment:
`export TESTTIMEOUT := $(ACCEPTANCETIMEOUT)` should do the same without the explicit `env` below!